vite.renderer.config.mts

```diff
@@ -1,16 +1,46 @@
 import react from '@vitejs/plugin-react';
+import fs from 'fs';
 import jotaiDebugLabel from 'jotai/babel/plugin-debug-label';
 import jotaiReactRefresh from 'jotai/babel/plugin-react-refresh';
 import path from 'path';
-import { HtmlTagDescriptor, defineConfig } from 'vite';
-import svgr from 'vite-plugin-svgr';
+import { HtmlTagDescriptor, defineConfig, loadEnv } from 'vite';
 import { createHtmlPlugin } from 'vite-plugin-html';
+import svgr from 'vite-plugin-svgr';
 
 import clientConfig from './client.config';
 
+const appVersion = Math.random().toString(36).substring(2, 15);
```

Comment:
I wonder if we should use the same version of package.json and github release.

Reply:
added this change here: [c2255c9](https://github.com/powerhouse-inc/connect/pull/476/commits/c2255c9c9830d285d23e798aa0d799a396e2bb7b)